### PR TITLE
Feature/disable select all for feature phones

### PR DESCRIPTION
**Pull request by moh-moola**

### CODE REVIEW

File: tuneme/static/new/js/main.js [hunk 1]
```diff
@@ -3,9 +3,17 @@
...
   domReady(function() {
     var selectRadius = document.getElementById('select_radius');
-    console.log(selectRadius[selectRadius.selectedIndex].value);
+    if(selectRadius){
+        console.log(selectRadius[selectRadius.selectedIndex].value);
+        // get selected option in sel (reference obtained above)
+        var optFunc = getSelectedOption(selectRadius);
+        selectRadius.addEventListener('click', function() {
+          console.log( optFunc.value," - ", optFunc.text);
+        });
```
Comment (moh-moola) [inline tuneme/static/new/js/main.js]: @Mitso is this piece of JS still relevant? can I remove it?

Comment (Mitso) [reply tuneme/static/new/js/main.js]: @moh-moola Please remove 👍

**Review by Mitso — APPROVED** [review]
👍 Is there a JQuery scripts loaded?
Make sure to load the js file when JS is enabled/supported or remove scripts when  JS is  not enabled/supported

**Comment by moh-moola** [comment]
@Mitso `Make sure to load the js file when JS is enabled/supported` what is the ideal way of doing this?

**Pull request merged**